src/Sarif.Driver.UnitTests/AnalyzeCommandTests.cs

```diff
@@ -209,6 +209,25 @@ public void LoadPdbException()
             );
         }
 
+
+        [Fact]
+        public void FileUri()
+        {
+            Uri uri = new Uri(this.GetType().Assembly.Location);
+
+            var options = new TestAnalyzeOptions()
+            {
+                TargetFileSpecifiers = new string[] { uri.ToString() },
+            };
+
+            ExceptionTestHelper(
+                ExceptionCondition.None,
+                RuntimeConditions.NoErrors,
+                analyzeOptions: options
```

Comment:
If the URI isn't accepted, we will return RuntimeConditions.NoValidTargets rather than RuntimeConditions.NoErrors.